gnovm/pkg/gnolang/gonative.go

```diff
@@ -1203,6 +1203,12 @@ func gno2GoValue(tv *TypedValue, rv reflect.Value) (ret reflect.Value) {
 			if ftv.IsUndefined() {
 				continue
 			}
+
+			// Skip unexported field
```

Comment:
Can you give more context on why you had to add this exception?

Reply:
Reflect panic when attempting to set a value on an unexported field. This prevents calling a method that returns a structure containing unexported fields. The typical approach is to skip unexported fields in this scenario. However, panic-ing may be justified in other contexts and if that the case I think we should have a way to call `Gno2Go` with an `skipUnexported` option to avoid panic and skip unexported.

Reply:
I need more explanation in the comment to understand why this addition was necessary, even though I see its relevance.

Reply:
I added more details in df0eb4f9ed75600c5cbae55f711ddcfff03c8cf2. Let me know if you find it sufficient